packages/docs/operations-manual/key-management.md

```diff
@@ -4,65 +4,209 @@
+
+
+## Summary
+
+The table below outlines a summary of the various key types in the Celo protocol. Note that these key types are *mutually exclusive*. A key that has been designated as one key type can never be used for a different purpose. For more details on a specific key type, please see that key type's specific section below.
+
+
+| Type                                | Description                                                                       | Ledger compatible |
+| ----------------------------------- |-----------------------------------------------------------------------------------|-------------------|
+| Undesignate                         | Default key type, cannot participate in proof-of-stake                            | Yes               |
+| Locked Gold Account (LGA)           | Used to lock and unlock cGLD and authorize other keys                             | Yes               |
+| Authorized vote signer              | Can vote on behalf of an LGA                                                      | Yes               |
+| Authorized validator (group) signer | Can register and manage a validator group on behalf of an LGA                     | No                |
+| Authorized validator signer         | Can register, manage a validator, and sign consensus messages on behalf of an LGA | No                |
+| Authorized validator BLS signer     | Used to sign blocks as a validator                                                | No                |
+| Authorized attestations signer      | Can sign attestations messages on behalf of a Locked Gold account                 | No                |
+
 
 {% hint style="warning" %}
-An account may have at most one authorized signer key of each type at any time. Once a signer key is authorized, the only way to deauthorize that key is to authorize a new key that has never previously been used as a signer key, either for this account or another. It follows then that a newly deauthorized key cannot be reauthorized for any account.
+A Locked Gold Account may have at most one authorized signer key of each type at any time. Once a signer key is authorized, the only way to deauthorize that key is to authorize a new key that has never previously been used as an authorized signer key for any Locked Gold Account. It follows then that a newly deauthorized key cannot be reauthorized for any account.
 {% endhint %}
 
-## Key Security
+## Undesignated
+
+By default, any private key generated in the Celo protocol does not have a specific role in proof-of-stake. These keys can be used to sign and send transactions that are unrelated to Celo's proof-of-stake protocol, including but by no means limited to transferring Celo Gold, transferring Celo Dollars, exchanging Celo Gold for Celo Dollars, etc.
+
+Undesignated keys can be designated as Locked Gold Accounts or authorized as signer keys on behalf of a Locked Gold Account by sending special transactions using `celocli`.
```

Comment:
I know  this is mostly written with respect to PoS, but the Account on `Accounts` isn't technically tied to LockedGold? I.e. for mobile wallets, you can create accounts to that one can set encryption keys. Smart contract developers could set an account to upload metadata to indicate trustworthiness.

Reply:
Very true. I'm not sure how to easily explain that nuance, which is not terribly relevant for the intended audience, without making things more confusing..